### ISSUE TITLE
Do we have NER dataset to load?

### ISSUE THREAD

**Issue by sakinaljana** [tensorflow/datasets#2692]
**What I need help with / What I was wondering**
Do we have example of creating NER/sequence labeling dataset? What are the preferrable approach to define the TFDS features? 

**What I've tried so far**
Already look at the documentation but can't find any sequence labeling dataset

**It would be nice if...**
I think we should have more identifier for the task for each dataset, so we can easily filter which dataset to use for what task.

**Comment by Conchylicultor** [comment]
What do you mean by sequence labelling ? Would `Sequence(ClassLabel(names=['label0',...]))` works ?

**Comment by sakinaljana** [comment]
@Conchylicultor seems so, my confusion more about the token, but at the end my features would be like this

```python
tfds.features.FeaturesDict({
                "tokens": tfds.features.Sequence(tfds.features.Text()),
                "tags": tfds.features.Sequence(tfds.features.ClassLabel(names=TAG_NAMES))
})
```

wdyt?

**Comment by Conchylicultor** [comment]
I'm not sure I understand your issue. Could you rephrase please ? Or close the issue if your issue has been solved.

**Comment by sakinaljana** [comment]
@Conchylicultor My question would be: What is the best features definition to represent NER dataset? Are my above definition is correct? Or do you have better suggestion?

**Comment by NikhilBartwal** [comment]
Hey @sakinaljana, looking at the NER dataset [here](https://www.kaggle.com/abhinavwalia95/entity-annotated-corpus), the following approach suggested by you should be working for the dataset features. 
> ```python
> tfds.features.FeaturesDict({
>                 "tokens": tfds.features.Sequence(tfds.features.Text()),
>                 "tags": tfds.features.Sequence(tfds.features.ClassLabel(names=TAG_NAMES))
> })
> ```
> 
For reference, you can have a look at the **helloswag** dataset [here](https://www.tensorflow.org/datasets/catalog/hellaswag). Did this solve your query?

**Comment by sakinaljana** [comment]
Thanks @NikhilBartwal, yes i think so. I'll close this issue
